Sources/Testing/Events/TimeValue.swift

```diff
@@ -8,7 +8,7 @@
 // See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 //
 
-@_implementationOnly import TestingInternals
+internal import TestingInternals
```

Comment:
If imports are `internal` by default, do we need this explicit `internal`? (And any others.)

Reply:
I was open to taking this suggestion, but just tried it and realized the compiler requires it:

> error: ambiguous implicit access level for import of 'TestingInternals'; it is imported as 'private' elsewhere